service/src/main/java/bio/terra/profile/app/controller/ProfileApiController.java

```diff
@@ -77,6 +75,37 @@ public ResponseEntity<Void> deleteProfile(@PathVariable("profileId") UUID id) {
     return new ResponseEntity<>(HttpStatus.NO_CONTENT);
   }
 
+  @Override
+  public ResponseEntity<SamPolicyModelList> getProfilePolicies(@PathVariable("profileId") UUID id) {
+    AuthenticatedUserRequest user = authenticatedUserRequestFactory.from(request);
```

Comment:
`AuthenticatedUserRequest` is deprecated, but we use it all through this codebase. Do we have a plan to change to `BearerToken`?

Reply:
https://broadworkbench.atlassian.net/browse/WOR-510